Lib/fontTools/t1Lib/__init__.py

```diff
@@ -367,3 +513,72 @@ def stringToLong(s):
 	for i in range(4):
 		l += byteord(s[i]) << (i * 8)
 	return l
+
+
+# PS stream helpers
+
+font_dictionary_keys = copy.copy(_type1_pre_eexec_order)
```

Comment:
no need to use the copy module here, you can just do `list(_type1_pre_eexec_order)`

Reply:
OK, I will use `list()`. I checked the [Python 3.7 documents](https://docs.python.org/3.7/library/stdtypes.html#typesseq-list) just to be sure, and it says that list() will make a copy if _iterable_ is already a list.